packages/example-app-base/src/components/ExampleCard/ExampleCard.styles.ts

```diff
@@ -39,11 +40,14 @@ export const getStyles: IStyleFunction<IExampleCardStyleProps, IExampleCardStyle
   };
 
   const codeButtonActiveStyles: IRawStyle = {
-    backgroundColor: theme.palette.neutralDark,
-    borderColor: theme.palette.neutralDark,
+    backgroundColor: NeutralColors.gray20,
+    borderColor: NeutralColors.gray90,
     selectors: {
-      '.ms-Button-icon, .ms-Button-label': {
-        color: theme.palette.white
+      '.ms-Button-icon': {
+        color: '#0078d4' // communicationPrimary
```

Comment:
Why hard code the hex instead of using `props.theme.palette.themePrimary`?

Reply:
Oh, I didn't realize it was the same color.